openquake/qa_tests_data/event_based_risk/case_2/expected/rlz-000-structural-loss-curves.csv

```diff
@@ -1,4 +1,5 @@
+a0,0.000000E+00 1.263357E+02 2.526715E+02 3.790072E+02 5.053430E+02 6.316787E+02 7.580144E+02 8.843502E+02 1.010686E+03 1.137022E+03 1.263357E+03 1.389693E+03 1.516029E+03 1.642365E+03 1.768700E+03 1.895036E+03 2.021372E+03 2.147708E+03 2.274043E+03 2.400379E+03,2.591818E-01 2.591818E-01 2.211992E-01 1.812692E-01 1.392920E-01 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 0.000000E+00,2.272640E+02
+a1,0.000000E+00 5.895119E+01 1.179024E+02 1.768536E+02 2.358047E+02 2.947559E+02 3.537071E+02 4.126583E+02 4.716095E+02 5.305607E+02 5.895119E+02 6.484630E+02 7.074142E+02 7.663654E+02 8.253166E+02 8.842678E+02 9.432190E+02 1.002170E+03 1.061121E+03 1.120073E+03,4.230502E-01 4.230502E-01 3.623718E-01 3.296800E-01 2.953119E-01 2.211992E-01 2.211992E-01 1.812692E-01 1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 0.000000E+00,2.096995E+02
+a2,0.000000E+00 4.687013E+01 9.374025E+01 1.406104E+02 1.874805E+02 2.343506E+02 2.812208E+02 3.280909E+02 3.749610E+02 4.218311E+02 4.687013E+02 5.155714E+02 5.624415E+02 6.093116E+02 6.561818E+02 7.030519E+02 7.499220E+02 7.967922E+02 8.436623E+02 8.905324E+02,1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 9.516258E-02 4.877058E-02 0.000000E+00,9.171676E+01
+a3,0.000000E+00 5.451772E+01 1.090354E+02 1.635531E+02 2.180709E+02 2.725886E+02 3.271063E+02 3.816240E+02 4.361417E+02 4.906594E+02 5.451772E+02 5.996949E+02 6.542126E+02 7.087303E+02 7.632480E+02 8.177657E+02 8.722835E+02 9.268012E+02 9.813189E+02 1.035837E+03,1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 1.392920E-01 9.516258E-02 9.516258E-02 9.516258E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 4.877058E-02 0.000000E+00,8.885412E+01
+a4,0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00,0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00,0.000000E+00
```

Comment:
One column less because we are not exporting the standard deviation, to be consistent with the format of other calculators (but we could restore the column if needed). One line more to be consistent with the engine, that lists all specified assets, even the ones with zero losses.